openwisp_utils/admin_theme/templates/admin/edit_inline/help_text_stacked.html

```diff
@@ -26,16 +27,18 @@ <h3><b>{{ inline_admin_formset.opts.verbose_name|capfirst }}:</b> <span class="i
   {% comment %} OpenWISP customization begin {% endcomment %}
   {% if inline_admin_formset.formset.help_text %}
     {% with inline_admin_formset.formset.help_text as help_text %}
-        <div class="ow-help-text">
-            <img src="{{ help_text.image_url|default:'/static/admin/img/icon-alert.svg' }}">
-            <p>{{ help_text.text}}
-            {% if help_text.documentation_url %}
-              {% blocktrans with anchor_open='<a href="'|add:help_text.documentation_url|add:'">'|safe anchor_close='</a>'|safe %}
-                Please refer to the {{ anchor_open }}documentation{{ anchor_close }} for more information.
-              {% endblocktrans %}
-            {% endif %}
-            </p>
-        </div>
+      {% with help_text.image_url|default:'/admin/img/icon-alert.svg' as icon_url %}
+          <div class="ow-help-text">
+              <img src="{% static icon_url %}">
```

Comment:
I just noticed something odd here, which was not introduced by you but was probably already there, but since we are at it it would be great to fix it now, do you see that the indentation level here is not consistent? It's 4 spaces while we are using two.

Reply:
@nemesisdesign I found [djhtml](https://github.com/rtts/djhtml) (Django/Jinja template indenter) with lots of available options. We can integrate this with `openwisp-utils` [(openwisp-qa-format script)](https://github.com/openwisp/openwisp-utils/blob/master/openwisp-qa-format) What do you think?

Reply:
2. Linter & Formatter : https://github.com/Riverside-Healthcare/djlint

Reply:
Sounds interesting, let's open an issue, although low priority for now :)

Reply:
Opened : https://github.com/openwisp/openwisp-utils/issues/315